CONTRIBUTING.md

```diff
@@ -97,6 +98,22 @@ All of the tests work offline. None of them should hit the network.
 $ npm test
 ```
 
+### Running only a single test
+
+When you're writing a test for a specific purpose, it may be useful to only run that test. In order to do this, you can pass a `only` option as the second argument to the [`tap`](https://www.node-tap.org/) testrunner. For instance in `tests/example.js`, like so:
+
+```js
+test('description', {only: true}, function(t) {
```

Comment:
Oh I think we use arrow function in most tests now, maybe change it to `(t) => {}`?